charts/tekton-apps/templates/argocd_projects.yaml

```diff
@@ -22,6 +22,10 @@ metadata:
   namespace: argo-cd
   annotations:
     argocd.argoproj.io/sync-wave: {{ $project.argocd.syncWave | default "200" | quote }}
+    {{- if $project.argocd.notifications }}
+    jiraURL: {{ $project.jiraURL }}
```

Comment:
This looks incorrect, so in cases when project has no `argocd.notifications`, you won't add `jiraURL` to annotations

Reply:
Yes, since that annotation was added specifically to be used in the notifications template, didn't wan't to unnecessarily add where isn't used.

I'll revert back to always add it